_posts/2025-05-09-opentelemetry-zero-code-instrumentation.md

```diff
@@ -0,0 +1,537 @@
+---
+title: "Zero-code Instrumentation for OpenFaaS Functions with OpenTelemetry"
```

Comment:
`Zero-code Instrumentation for OpenFaaS Functions with OpenTelemetry` - I think since this is the first time we've walked about OTEL officially, we can go for a more generic title.

"How to Trace Functions with OpenTelemetry"